docs/user_guide/assessment.rst

```diff
@@ -138,6 +140,9 @@ metrics simultaneously:
     c                         0.6   0.75
     d                         0.0    0.0
 
+Note that :func:`fairlearn.metrics.count` can be used to show the number of subjects in
```

Comment:
Subjects sounds like we're doing an experimental trial of sorts 🙂 How about people / rows? Or simply group size?

Reply:
Yeah, that makes sense. I'll get that changed to group size.

Reply:
Did you want to change this still?